### PR TITLE
Added liked videos page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Added the liked videos page : 

- The user can add any video to the liked page 
- The use can remove the video from the liked page
Screenshot (three)

### CODE REVIEW

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
+import axios from "axios";
 import { VideoCard } from "../../Components";
+import { useAuth } from "../../Contexts/Authcontext";
 import { useData } from "../../Contexts/Datacontext";
 import "./SingleVideo.css";
 
 const SingleVideo = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const { dispatch } = useData();
```
Comment (SJTGSHIVAM) [inline TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: dont call twice please extract all vars st once

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx [hunk 2]
```diff
@@ -1,15 +1,65 @@
...
+  const { token } = useAuth();
   const { state } = useData();
```
Comment (SJTGSHIVAM) [inline TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: here is second

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx [hunk 3]
```diff
@@ -1,15 +1,65 @@
...
+  const { liked } = state;
```
Comment (SJTGSHIVAM) [inline TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: same

**Review by SJTGSHIVAM — APPROVED** [review]
can remove console logs once debugging is done.

File: TravelBug/src/Components/VerticalCard/VerticalCard.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
+.vertical-card-container {
+  padding: 1rem;
+  background-color: rgb(205, 205, 205);
```
Comment (mdarif6) [inline TravelBug/src/Components/VerticalCard/VerticalCard.css]: - put color variable

File: TravelBug/src/Components/VerticalCard/VerticalCard.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+.modal-child {
+  margin-top: 1rem;
+  border: 1px solid black;
+  padding: 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--white);
+  border-radius: 0.2rem;
+  font-family: var(--card-font);
```
Comment (mdarif6) [inline TravelBug/src/Components/VerticalCard/VerticalCard.css:44]: - here too put color variable

File: TravelBug/src/Components/VerticalCard/VerticalCard.jsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  };
+
+  const deleteLikeHandler = async () => {
+    try {
+      const res = await axios.delete(`/api/user/likes/${_id}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log("del response ", res);
```
Comment (mdarif6) [inline TravelBug/src/Components/VerticalCard/VerticalCard.jsx]: - you can remove this console, i think it was for testing purpose only

**Review by mdarif6 — APPROVED** [review]
- rest is seems okay

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (Shah-Arsalan) [reply TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: Done

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx (hunk 2 above)
Comment (Shah-Arsalan) [reply TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: Done

File: TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx (hunk 3 above)
Comment (Shah-Arsalan) [reply TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx]: Done

File: TravelBug/src/Components/VerticalCard/VerticalCard.css (hunk 4 above)
Comment (Shah-Arsalan) [reply TravelBug/src/Components/VerticalCard/VerticalCard.css]: done

File: TravelBug/src/Components/VerticalCard/VerticalCard.jsx (hunk 6 above)
Comment (Shah-Arsalan) [reply TravelBug/src/Components/VerticalCard/VerticalCard.jsx]: Done

**Pull request merged**